pkg/registry/apis/provisioning/resources/parser.go

```diff
@@ -223,11 +229,15 @@ func (r *Parser) Parse(ctx context.Context, info *repository.FileInfo, validate
 }
 
 func (f *ParsedResource) ToSaveBytes() ([]byte, error) {
-	// TODO? should we use the dryRun (validated) version?
```

Comment:
🤔 I don't think so -- the intention with deep copy is to see what would happen afterwards, but the input is the same

The difference would be:
* any metadata/status fields (but removed anyway)
* if mutations happen... the dryRun will include them